po/common/en_GB.po

```diff
@@ -1094,7 +1094,7 @@ msgstr "Products that do not contain the ingredient %s"
 # Please change appstore_US.svg to appstore_XX.svg. check the url https://static.openfoodfacts.org/images/misc/appstore/black/appstore_XX.svg
 msgctxt "ios_app_icon_url"
 msgid "/images/misc/appstore/black/appstore_US.svg"
-msgstr "/images/misc/appstore/black/appstore_UK.svg"
+msgstr "/images/misc/appstore/black/appstore_US.svg"
```

Comment:
Change doesn't seem necessary, because [appstore_UK.svg](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/html/images/misc/appstore/black/appstore_UK.svg) exists?